cli/cmd/project/members.go

```diff
@@ -0,0 +1,195 @@
+			}
+
+			client, err := client.New(cfg.AdminURL, cfg.AdminToken())
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+			_, err = client.AddProjectMember(cmd.Context(), &adminv1.AddProjectMemberRequest{
+				Organization: orgName,
+				Project:      args[0],
+				Email:        args[1],
+				Role:         args[2],
+			})
+			if err != nil {
+				return err
+			}
+			cmdutil.SuccessPrinter("Done")
+			return nil
+		},
+	}
+	return addCmd
+}
+
+func RemoveCmd(cfg *config.Config) *cobra.Command {
+	removeCmd := &cobra.Command{
+		Use:   "remove",
+		Short: "Remove Member",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			orgName := cfg.Org()
+			if orgName == "" {
+				return errors.New("cannot not find default org")
+			}
+			if len(args) != 2 {
+				return errors.New("invalid number of arguments (expected 2) - ProjectName Email")
+			}
+
+			client, err := client.New(cfg.AdminURL, cfg.AdminToken())
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+			_, err = client.RemoveProjectMember(cmd.Context(), &adminv1.RemoveProjectMemberRequest{
+				Organization: orgName,
+				Project:      args[0],
+				Email:        args[1],
+			})
+			if err != nil {
+				return err
+			}
+			cmdutil.SuccessPrinter("Removed")
+			return nil
+		},
+	}
+	return removeCmd
+}
+
+func SetRoleCmd(cfg *config.Config) *cobra.Command {
+	setRoleCmd := &cobra.Command{
+		Use:   "set-role",
+		Short: "Set role of Member",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			orgName := cfg.Org()
+			if orgName == "" {
+				return errors.New("cannot not find default org")
+			}
+			if len(args) != 3 {
+				return errors.New("invalid number of arguments (expected 2) - ProjectName Email Role")
+			}
+
+			client, err := client.New(cfg.AdminURL, cfg.AdminToken())
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+			_, err = client.SetProjectMemberRole(cmd.Context(), &adminv1.SetProjectMemberRoleRequest{
+				Organization: orgName,
+				Project:      args[0],
+				Email:        args[1],
+				Role:         args[2],
+			})
+			if err != nil {
+				return err
+			}
+			cmdutil.SuccessPrinter("Updated")
+			return nil
+		},
+	}
+	return setRoleCmd
+}
+
+func toUserTable(users []*adminv1.User) []*user {
+	allUsers := make([]*user, 0, len(users))
+
+	for _, usr := range users {
+		allUsers = append(allUsers, toUserRow(usr))
+	}
+
+	return allUsers
+}
+
+func toUserRow(u *adminv1.User) *user {
+	return &user{
+		Name:      u.DisplayName,
+		Email:     u.Email,
+		CreatedOn: u.CreatedOn.AsTime().String(),
+		UpdatedOn: u.UpdatedOn.AsTime().String(),
+	}
+}
+
+type user struct {
+	Name      string `header:"name" json:"display_name"`
+	Email     string `header:"email" json:"email"`
+	CreatedOn string `header:"created_on,timestamp(ms|utc|human)" json:"created_on"`
+	UpdatedOn string `header:"updated_on,timestamp(ms|utc|human)" json:"updated_on"`
+}
```

Comment:
Move these to `cmdutil` to prevent duplication. Could add `cmdutil.PrintUsers`, etc.